api/controller.py

```diff
@@ -1349,13 +1366,6 @@ def sync(self):
         if isinstance(response, Response):
             return response
 
-        # TODO: SimplyE used to make a HEAD request to the bookshelf feed
-        # as a quick way of checking authentication. Does this still happen?
-        # It shouldn't -- the patron profile feed should be used instead.
-        # If it's not used, we can take this out.
-        if flask.request.method=='HEAD':
```

Comment:
Since this is being removed, should `HEAD` be removed from [the available methods](https://github.com/NYPL-Simplified/circulation/blob/develop/api/routes.py#L438) for this endpoint?

Reply:
Nevermind, I see that you've already removed it now, but it should probably retain the `GET` method.